src/site/content/en/blog/monitor-total-page-memory-usage/index.md

```diff
@@ -255,59 +210,60 @@ async function performMeasurement() {
 }
 ```
 
+Finally, begin measuring.
+
+```javascript
+// Start measurements.
+scheduleMeasurement();
+```
+
 The result may look as follows:
 
 ```javascript
 // Console output:
 {
-  bytes: 60_000_000,
+  bytes: 60_100_000,
```

Comment:
Only nit comment is it's not clear why this changed.

Reply:
it's because there's the new `types: ['DOM']` entry below with 100,000 bytes. I could just make it 10,000,000 bytes of DOM and the total could be an even 70,000,000 if the value is distracting?

Reply:
Ahh, that's right (you explained this to me before).

I think it's only distracting in the diff; I think it won't be distraction to someone viewing the post as a whole.